### ISSUE TITLE
How to get the output of the network in the same format the input labels were given in the json? 

### ISSUE THREAD

**Issue by TheBlackCat22** [dairui01/MS-TCT#6]
How do i convert the logits i get out of the network into the format of the actions in the input json (Action, Start_time, End_time)?

**Comment by dairui01** [comment]
Our evaluation of the datasets is the frame-level evaluation following the original setting of the dataset. For this reason, we need to generate the frame-level ground truth from the JSON file and used them for network training and testing. The output of our model should be the frame-level action scores rather than the action events. 

For generating the event-level logits from the model, you need to change the last prediction head of the model to the event-level prediction head (i.e., regression head for predicting the start/end or action duration + a classifier for predicting action category). The training ground truth will be the action event (i.e., the one provided directly in the JSON). Some existing attempts that utilize MSTCT for action event prediction can be found [here](https://arxiv.org/pdf/2210.11035.pdf).
